### DISCUSSION TITLE
Include `orders::currency_value` when displaying an order's details?

### DISCUSSION THREAD

**Discussion by lat9** [zencart/zencart#6078] [Ideas]
If an order is placed in a currency _other than_ the site's default currency, the exchange rate used in the currency calculations is stored in the subject database field, but is never displayed to the admin user.

Shouldn't that value be displayed if it's different than the default?

**Comment by drbyte** [comment]
I see your point, and agree:  If there's a variance from defaults then it's sensible to disclose that if it will help the rest of what's displayed make sense.

**Comment by brittainmark** [comment]
I agree i would always display the exchange rate if it is not the default currency.

**Comment by lat9** [comment]
I'll note also that a store's default currency, which is the basis for the `currency_value` isn't recorded in the order either.

Consider, for example, a store in Spain which (until 2002) use the pesata as its default currency and now uses the euro.  Since the `currency_value` is a multiplier based on the store's default currency _at the time the order is placed_, that value might not have a valid basis.
